packages/tooltip/src/Tooltip.tsx

```diff
@@ -252,14 +277,20 @@ function Tooltip({
     enabled: open && triggerEvent === 'click',
   });
 
+  const portalProps = { usePortal, portalClassName } as Pick<
+    PopoverProps,
+    'usePortal' | 'className'
+  >;
+
   const tooltip = (
     <Popover
+      key="tooltip"
       active={enabled && open}
       align={align}
       justify={justify}
-      usePortal={usePortal}
       adjustOnMutation={true}
       spacing={12}
+      {...portalProps}
```

Comment:
micro perf nit: seems un-necessary to create another object here, can we just directly pass the props directly?

Reply:
from a typing angle we can't because once we've destructured the props out, they lose the "OneOf"iness. we create another object in order to get to get the "OneOf"iness back.